__tests__/app.test.js

```diff
@@ -149,4 +159,41 @@ describe("/api/articles/:article_id/comments", () => {
             expect(body.msg).toBe("Article not found");
         });
     })
+    test("POST: 201 responds with the posted comment with all neccessary properties", () => {
+        const testComment = { username: "butter_bridge", body: "the war is finally over" };
+        return request(app)
+        .post("/api/articles/12/comments")
+        .send(testComment)
+        .expect(201)
+        .then((response) => {
+            expect(response.body.comment).toMatchObject({
+                comment_id: expect.any(Number),
+                article_id: 12,
+                author: 'butter_bridge',
+                body: 'the war is finally over',
+                votes: 0,
+                created_at: expect.any(String),
+            });
+        });
+    });
+    test("POST: 400 responds with an error when the request body is missing a username or body field", () => {
+        const testComment = { body: "the war is finally over" };
+        return request(app)
+            .post("/api/articles/12/comments")
+            .send(testComment)
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe("Bad Request");
+            });
+    });
+    test("POST: 404 responds with an error when the article_id does not exist", () => {
+        const testComment = { username: "masterChief", body: "the war is finally over" };
+        return request(app)
+        .post("/api/articles/999/comments")
+        .send(testComment)
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("Article not found");
+        });
+    });
```

Comment:
Maybe have a test for ignores additional keys in the body, otherwise really good work